src/extensions/views/InstalledExtensions/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { InstalledExtension } from "@dashboard/extensions/types";
+import { PluginBaseFragment } from "@dashboard/graphql";
+
+export const byActivePlugin = (plugin: PluginBaseFragment) => {
+  return (
+    plugin?.globalConfiguration?.active === true ||
+    plugin?.channelConfigurations?.some(config => config.active)
+  );
+};
+
+export const sortByName = (a: InstalledExtension, b: InstalledExtension) => {
+  if (a.name.toLowerCase() > b.name.toLowerCase()) {
+    return 1;
+  }
+
+  if (a.name.toLowerCase() < b.name.toLowerCase()) {
+    return -1;
+  }
+
+  return 0;
+};
```

Comment:
suggestion: we could replace it with `String.localeCompare()` with options to ignore case: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator#ignorepunctuation